src/test/java/frc/robot/subsystems/example/ShooterTest.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems.example;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import edu.wpi.first.units.Units;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.ShooterIO;
+import frc.robot.subsystems.shooter.ShooterIOSim;
+import org.junit.Test;
+
+public class ShooterTest implements AutoCloseable {
+    private final Shooter shooter;
+
+    public ShooterTest() {
+        ShooterIO shooterIO = new ShooterIOSim();
+        Shooter.initialize(shooterIO);
+        shooter = Shooter.getInstance();
+    }
+
+    @Test
+    public void testSetVelocity() throws InterruptedException {
+        shooter.setDefaultCommand(
+                shooter.setVelocity(() -> Units.RotationsPerSecond.of(100).mutableCopy()));
+        Thread.sleep(3000);
```

Comment:
Setting it as the default command won't do anything if the line `CommandScheduler.getInstance().run()` is not called, which is done in the robot periodic. I think just adding that before sleeping the thread would do the trick.